test/tic_tac_clojure/ai_test.clj

```diff
@@ -0,0 +1,35 @@
+(ns tic-tac-clojure.ai-test
+  (:require [clojure.test :refer :all]
+            [tic-tac-clojure.ai :refer :all]
+            [tic-tac-clojure.sample-boards :refer :all]))
+            
+(def sample-scores-map-1
+  {2 1
+   3 -1 
+   4 -1
+   7 1
+   8 -5})
+
+(def sample-scores-map-2
+  {0 -1
+   3 1 
+   4 5
+   7 1
+   8 -5})
+
+(deftest get-opp-marker-test
```

Comment:
In this namespace, I've made sure to only make `choose-best-space` publicly available. 

How does TDD work with complex functions that can/should be broken down into helper methods? I can see `choose-best-space` being one very long function that implements minimax under the hood. I'm rarely able to get a complex function correct on the first try, so I usually try to write the helper methods with tests so that I can build up to the final function. Should I be writing tests for helper methods then deleting them when the public-facing method passes its own tests?